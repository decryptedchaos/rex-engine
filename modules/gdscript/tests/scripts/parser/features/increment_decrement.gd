func test():
	var a = 100
	var b = 5000

	print(a++)
	print(++a)
	print(a--)
	print(--a)

	# (a++) + b
	var c = a+++b

	print(a)
	print(c)

	c = --a + b
	print(a)
	print(c)
