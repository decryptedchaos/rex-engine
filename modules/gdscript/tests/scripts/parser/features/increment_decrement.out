GDTEST_OK
100
102
102
100
101
5100
100
5100
